src/__internal__/character-count/character-count.spec.tsx

```diff
@@ -17,13 +21,75 @@ describe("CharacterCount", () => {
     it("should render default", () => {
       assertStyleMatch(
         {
-          textAlign: "right",
+          textAlign: "left",
           fontSize: "12px",
           marginTop: "4px",
           marginBottom: "4px",
           color: "var(--colorsUtilityYin055)",
         },
-        wrapper
+        wrapper.find(StyledCharacterCount)
+      );
+    });
+
+    it("character counter has aria-hidden attribute", () => {
+      expect(wrapper.find(StyledCharacterCount).prop("aria-hidden")).toBe(
+        "true"
+      );
+    });
+
+    it("character counter has correct count", () => {
+      expect(wrapper.find(StyledCharacterCount).text()).toBe(
+        "5 characters left"
+      );
+    });
+
+    it("visually hidden character counter has aria-live='polite'", () => {
+      expect(wrapper.find(VisuallyHiddenCharacterCount).prop("aria-live")).toBe(
+        "polite"
+      );
+    });
+
+    it("visually hidden character counter has the same count message", () => {
+      expect(wrapper.find(VisuallyHiddenCharacterCount).text()).toBe(
+        "5 characters left"
+      );
+    });
+
+    it("visually hidden character counter has visuallyHidden class applied", () => {
```

Comment:
**suggestion(non-blocking)**: likewise here, since we aren't importing the `visuallyHidden` class for use inside the test, this might help another developer understand the context behind this test if we decide amend/remove the `visuallyHidden` util in the future.

```suggestion
    it("visually hidden character counter has css properties to visually hide it from view", () => {
```